dev-docs/RFCs/v6.x/gpu-aggregation-enhancements.md

```diff
@@ -0,0 +1,112 @@
+# RFC: GPU Aggregation enhancements
+
+* **Authors**: Ravi Akkenapally
+* **Date**: Sep 2018
+* **Status**: **Review**
+
+## Problem
```

Comment:
Don't forget the 2-3 line "Summary": This RFC proposes ...

I think you basically have the right text in the "Problem" Section

Reply:
Updated the text.